src/blocks/components/form-error-message-wrapper/form-error-message-wrapper-style.scss

```diff
@@ -0,0 +1,5 @@
+.form-error-message-wrapper {
+  &.hide-form-message {
```

Comment:
Why not `.is-hidden`?

Reply:
Because previously when discussing as similar PR with @iruzevic he metioned we should probably use specific classes for this for components (rather than something general as `is-hidden`). I'm kinda fine with both 🤷‍♂️

Reply:
Dunno, I thought that we always used the same 'helper' css classes for these kinds of things 🤷‍♂️ 

I'm fine with both, just thought the default was like `.is-{action}` 🤷‍♂️

Reply:
We should discuss this on the next Protein / standup, I agree we should be consistent. Also makes sense for `is-{action}` format, I'll fix this.